R/addins.R

```diff
@@ -91,6 +91,13 @@ style_active_pkg <- function() {
 #' @importFrom rlang abort
 #' @keywords internal
 style_selection <- function() {
+  if (rstudio_has_selection_apis())
+    style_selection_v2()
+  else
+    style_selection_v1()
+}
+
+style_selection_v1 <- function() {
```

Comment:
Why v1 and v2? Does this refer to the `rstudioapi` major version?

Reply:
Just arbitrary versions to indicate one is "newer" and one is "older".